### PR TITLE
Add YAML file for variable defaults and add new variables

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
Closes Issues #35 and #44.

This pull request removes the Python dictionary that defines variable defaults such as colorbar, vmin, vmax, etc. and replaces it with a default YAML file that can be added to. The user is also free to pass a path to their own YAML file if they choose.

### CODE REVIEW

File: src/emcpy/plots/variable_specs.py [hunk 1]
```diff
@@ -1,78 +1,23 @@
...
-        'diff contour interval': 0.5,
-        'cmap': 'rainbow'
-    }
-}
+import yaml
+from pathlib import Path
+import os
 
 
 class VariableSpecs:
```
Comment (CoryMartin-NOAA) [inline src/emcpy/plots/variable_specs.py:7]: @kevindougherty-noaa not in this PR but we probably need documentation for this class?

Comment (kevindougherty-noaa) [reply src/emcpy/plots/variable_specs.py:7]: Yes, I agree.

**Review by kevindougherty-noaa — DISMISSED** [review]
Looks good to me.

**Review by kevindougherty-noaa — APPROVED** [review]

**Review by RussTreadon-NOAA — APPROVED** [review]
hofxcs `feature/emcpy_plotting` has been updated to [effad006](https://github.com/NOAA-EMC/hofxcs/commit/effad006de507b028a99ec14533fc1589d07b7f6).    emcpy `feature/var_defaults_yaml` has been updated to 0756c8d7.   The combination of these two have successfully generated plots for 4 cycles on Orion.   From this perspective both `feature/emcpy_plotting` and `feature/var_defaults_yaml` should be merged into the develop branch in their respective repositories.

**Pull request merged**